tests/scanners/test.json.js

```diff
@@ -20,11 +18,7 @@ describe('JSONScanner', () => {
       return Promise.reject(new Error('Explode!'));
     });
 
-    return jsonScanner.scan()
-      .then(unexpectedSuccess)
-      .catch((err) => {
-        expect(err.message).toEqual('Explode!');
-      });
+    await expect(jsonScanner.scan()).rejects.toThrow('Explode!');
   });
 
   it('should use special parser for messages.json', () => {
```

Comment:
This test was added in a PR from me and should probably also be refactored.

Reply:
@freaktechnik Thanks for pointing it out! ❤️ 

@hameleonka do you mind to refactor also this new test case in this PR?

Reply:
@rpl done 👍